aspose/src/test/java/nl/nn/adapterframework/extensions/aspose/pipe/PdfPipeTest.java

```diff
@@ -126,6 +130,7 @@ public void expectSuccessfullConversion(String pipeName, String fileToConvert, S
 
 			PDFUtil pdfUtil = new PDFUtil();
 			pdfUtil.setCompareMode(CompareMode.VISUAL_MODE);
+			pdfUtil.setAllowedRGBDeviation(0.3); //In percents
```

Comment:
`0.3` was dus te klein. Het zou handig zijn als we op een of andere manier in de tests konden zien, wat de werkelijke deviation is zonder alle logs te moeten downloaden maar daarvoor zal wat refactoring nodig zijn om dat in de mogelijk te maken!

Reply:
Ja klopt, ik heb er overnagedacht, maar vond het iets te veel werk 😢 
Voor nu heb ik de logs maar gedownload, 0.4 bij veel tests en 1 test 0.51

Reply:
Een marge van 1 pct is dan misschien iets veiliger, maar voorlopig zou de build dan ook hiermee moeten werken.